tldw_Server_API/Config_Files/templates/watchlists/grouped_briefing_markdown.md

```
@@ -0,0 +1,59 @@
+---
+name: grouped_briefing_markdown
+format: md
+description: Grouped briefing with optional LLM summaries
+---
+# {{ title }}
+
+Generated: {{ generated_at }}
+Items: {{ item_count }}{% if has_groups %} | Groups: {{ group_count }}{% endif %}
+
+{% if has_briefing_summary %}
+## Executive Summary
```

markdownlint-cli2 (0.21.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Above

(MD022, blanks-around-headings)